vmm/src/api/http.rs

```diff
@@ -196,7 +196,11 @@ fn handle_http_request(
             Ok(notifier) => route.handle_request(&request, notifier, api_sender.clone()),
             Err(_) => Response::new(Version::Http11, StatusCode::InternalServerError),
         },
-        None => Response::new(Version::Http11, StatusCode::NotFound),
+        None => {
+            let mut r = Response::new(Version::Http11, StatusCode::NotFound);
+            r.set_body(Body::new("Invalid request!\n"));
```

Comment:
I'm fine with adding a response body to be more explicit, but then it should be related to the fact that this is an unsupported endpoint. Something like ```"Unsupported endpoint: {}", path```

Reply:
None of our other errors return messages. And 404 (Not Found) already tells you that this in an invalid endpoint so I don' think you need to add a message in this case.

Reply:
Agreed, the message is redundant. *If* we were to add a message it would have to be different, but since none of our errors return a message, let's keep it that way for now.